src/api/spec/bootstrap/features/webui/requests_spec.rb

```diff
@@ -23,6 +23,23 @@
   end
 
   context 'request show page' do
+    let!(:superseded_bs_request) do
+      request = create(:set_bugowner_request)
```

Comment:
what about doing this what a factory? or there is any reason why this is not possible? :confused:

Reply:
That wouldn't work because the request model doesn't allow to initialize a request as superseded